demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -811,6 +814,14 @@ def get_modified_and_added_files(self, compare_type, prev_ver):
         """
         if not self.no_configuration_prints:
             click.echo("Collecting all committed files")
```

Comment:
move this print to line 814.
This serves as the header for the committed files check.